src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -38,6 +41,23 @@ public override JObject ToJson()
 
         public override bool Verify(DataCache snapshot, Transaction tx)
         {
+            // Check already stored signers
+
+            var conflictingSigners = tx.Signers.Select(s => s.Account);
+
+            foreach (var attr in tx.GetAttributes<Conflicts>())
+            {
+                var conflictRecord = snapshot.TryGet(
+                    new KeyBuilder(NativeContract.Ledger.Id, LedgerContract.Prefix_Transaction).Add(attr.Hash))?
+                    .GetInteroperable<TransactionState>();
+                if (conflictRecord == null) continue;
+
+                if (conflictRecord.ConflictingSigners.Concat(conflictingSigners).Distinct().Count() > 100)
+                {
+                    return false;
+                }
+            }
+
```

Comment:
The direction is right, this will fix the problem (if, of course, the true source of the problem is the conflicting signers list overflow). We need to test this patch against the nice scenario provided by @vang1ong7ang in https://github.com/neo-project/neo/issues/2907#issue-1898320065.